apps/web/src/pages/Applications/ApplicationWelcomePage/ApplicationWelcomePage.tsx

```diff
@@ -118,13 +119,21 @@ const ApplicationWelcome = ({ application }: ApplicationPageProps) => {
         )}
       </p>
       <p data-h2-margin="base(x1, 0)">
-        {intl.formatMessage({
-          defaultMessage:
-            "The GC Digital Talent platform is a skills-based hiring system. This means that your application will put a heavier focus on your skills and how you've used them in past experiences to help us get a stronger understanding of your fit.",
-          id: "u/DBSl",
-          description:
-            "Description of how the skills-based hiring platform assess candidates.",
-        })}
+        {isIAP
+          ? intl.formatMessage({
+              defaultMessage:
+                "The Program is a Government of Canada initiative specifically for First Nations, Inuit, and Métis peoples. It is pathway to employment in the federal public service for Indigenous peoples who have a passion for Information Technology (IT). We focus on that passion, and your potential to grow and succeed in this field.",
```

Comment:
- word **program** is not a proper noun and should be lowercase
- second sentence missing the indefinite article **a** before pathway

```suggestion
                "The program is a Government of Canada initiative specifically for First Nations, Inuit, and Métis peoples. It is a pathway to employment in the federal public service for Indigenous peoples who have a passion for Information Technology (IT). We focus on that passion, and your potential to grow and succeed in this field.",
```

@substrae please update the [figma](https://www.figma.com/file/8piNhFaBjcxssBB8LAGxEY/IAP-User-flows?type=design&node-id=269-76293&t=7ETAEU0FCLxI8awi-0) file accordingly.